src/GitExtensions.GerritPlugin/GitExtensions.GerritPlugin.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GitExtensions.Extensibility" Version="0.1.1.40" />
+  </ItemGroup>
+  <!-- Reference to GitExtensions dlls. -->
+  <ItemGroup>
+    <Reference Include="GitCommands">
+      <HintPath>$(GitExtensionsPath)\GitCommands.dll</HintPath>
+    </Reference>
+    <Reference Include="GitExtUtils">
+      <HintPath>$(GitExtensionsPath)\GitExtUtils.dll</HintPath>
+    </Reference>
+    <Reference Include="GitUI">
+      <HintPath>$(GitExtensionsPath)\GitUI.dll</HintPath>
+    </Reference>
+    <Reference Include="ResourceManager">
+      <HintPath>$(GitExtensionsPath)\ResourceManager.dll</HintPath>
+    </Reference>
+    <Reference Include="GitUIPluginInterfaces">
+      <HintPath>$(GitExtensionsPath)\GitUIPluginInterfaces.dll</HintPath>
+    </Reference>
+    <Reference Include="JetBrains.Annotations">
+      <HintPath>$(GitExtensionsPath)\Plugins\JetBrains.Annotations.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.VisualStudio.Threading">
+      <HintPath>$(GitExtensionsPath)\Microsoft.VisualStudio.Threading.dll</HintPath>
+    </Reference>
```

Comment:
@RussKie This connects to https://github.com/gitextensions/gitextensions/pull/7741#issuecomment-583797771:

It works, but I consider it rather fragile. A plugin should not depend on whether future GE versions keep packing the right version of `JetBrains.Annotations.dll` or `Microsoft.VisualStudio.Threading.dll`.

Obviously, a plugin will always depend on some internal library like `GitCommands.dll` or `GitExtUtils.dll`, but I see no need to have the same type of dependency for external libraries.
I prefer each plugin to contain it's own external dependencies, even if this means we will have the same DLL in multiple places: one for GE and one for the plugin.

Reply:
Good point, I agree.
Can we remove these?

Reply:
> Can we remove these?

Yes, will do.

Reply:
This is still outstanding, other than this looks good

Reply:
Done